src/js/env/browser/browserUtils.js

```diff
@@ -51,6 +51,12 @@ export const getBrowserState = (supportedBrowsers: {
     };
 };
 
+export const getOS = () => {
+    if (!navigator || !navigator.userAgent) return 'unknown';
+    const browser = Bowser.getParser(navigator.userAgent);
+    return browser.getOS().name.toLowerCase();
```

Comment:
`name` is defined as possibly undefined (probably could never happen, but better safe than sorry)

Reply:
addressed in 4270cee

also a lot of tests were added